src/applications/simple-forms/mock-simple-forms-patterns/sass/web-component-adapter.scss

```diff
@@ -23,4 +23,13 @@
 //     &::part(label) {
 //         font-weight: 600;
 //     }
-// }
\ No newline at end of file
+// }
+
+// Temporary styling to try styles that DST could apply
+// TODO - remove
```

Comment:
UX is still being decided regarding military checkbox. current solution is to change country to 'inert', which was a recommendation by DST.  This is sort of like disabled. - a read only field.